EventFilter/CTPPSRawToDigi/src/RawDataUnpacker.cc

```diff
@@ -80,11 +80,14 @@ int RawDataUnpacker::ProcessOptoRxFrame(const word *buf, unsigned int frameSize,
   #endif
 
   // parallel or serial transmission?
-  if (FOV == 1)
-    return ProcessOptoRxFrameSerial(buf, frameSize, fc);
-
-  if (FOV == 2)
-    return ProcessOptoRxFrameParallel(buf, frameSize, fedInfo, fc);
+  switch (FOV) {
+    case 1:
+      return ProcessOptoRxFrameSerial(buf, frameSize, fc);
+    case 2:
+    case 3:
```

Comment:
what does FOV=3 correspond to?
I do not see any other places where the code uses it to do something.

If this is for a firmware version that can have the compact format?